gov2/rekognition/README.md

```diff
@@ -0,0 +1,74 @@
+# AWS SDK for Go V2 code examples for Amazon Rekognition
+
+## Purpose
+
+These examples demonstrates how to perform several Amazon Rekognition
+operations using version 2 of the AWS SDK for Go.
+
+## Prerequisites
+
+You must have an AWS account, and have your default credentials and AWS Region
+configured as described in
+[Configuring the AWS SDK for Go V2](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/)
+in the AWS SDK for Go V2 Developer Guide.
+
+## Running the code
+
+### DetectLabels/DetectLabels.go
+
+This example performs three tasks:
+
+1. Saves the image in an Amazon Simple Storage Service (Amazon S3) bucket with a "uploads/" prefix.
```

Comment:
Again, this should be an "uploads/" prefix.

Reply:
Fixed